.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,11 @@
+name: "💡 Feature Request"
```

Comment:
what does this do?

Reply:
It is the same as the current template for suggesting a feature

Reply:
I meant, what are these YML files? How do they work?

Reply:
I was following [this documentation](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) on how to set up an issue form as I thought it would be useful to have separate parts such as page and line number etc. The YAML format allows it to be like a form rather than one text box.

I also used these issue pages for help/inspiration [freeCodeCamp_new_issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/new/choose), [PowerToys_new_issue](https://github.com/microsoft/PowerToys/issues/new/choose)

Reply:
I haven't tried this before. Let's merge it into `main` instead of `dev` and see if it works.

Reply:
ok